### ISSUE TITLE
Questions duplicate

### ISSUE THREAD

**Issue by iscmiguelsamaniego** [drozdzynski/Steppers#25]
Hi, i have a problem with different fragmen, every 2 questions duplicate item instead show the fragment declared, this happends when i return to fragment, any idea, thanks.

**Comment by drozdzynski** [comment]
Can you make some video / gif / screens?

**Comment by iscmiguelsamaniego** [comment]
Yes, i market the screen images, thanks and greetings from mexico.

https://drive.google.com/drive/folders/0BwtchwuVtpt5YzBzaHpXZFdURlk?usp=sharing
